src/hooks/useProject.js

```diff
@@ -52,6 +52,17 @@ export const useProject = ({
       );
       return;
     }
+
+    if (code) {
+      const project = {
+        name: "Blank project",
+        type: "python",
+        components: [{ name: "main", extension: "py", content: code }],
+      };
+      dispatch(setProject(project));
+      return;
+    }
```

Comment:
💭 might want to think about how we can expand this to HTML code too - not for this but a nice thing to think about 😁